.github/workflows/ci.yml

```
@@ -1017,6 +1017,104 @@ jobs:
 
+  ci-admin-smoke:
+    name: Admin E2E Smoke
+    environment: Preview – jovie
+    # Gated: only runs with 'testing' label (admin tests need auth credentials)
+    # Also runs on push to main for full coverage
+    needs: [ci-fast]
+    if: ${{ (github.event_name == 'push' && github.ref == 'refs/heads/main') || (github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork == false && contains(github.event.pull_request.labels.*.name, 'testing')) }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd # v6.0.2
```

YAMLlint (1.38.0)

[warning] too few spaces before comment: expected 2

(comments)